sklearn/utils/tests/test_extmath.py

```diff
@@ -720,3 +721,47 @@ def test_safe_sparse_dot_dense_output(dense_output):
     if dense_output:
         expected = expected.toarray()
     assert_allclose_dense_sparse(actual, expected)
+
+
+class TestingLinearOperator(LinearOperator):
```

Comment:
Let's call this  `CustomLinearOperator` or something similar, `Test.*` in the name suggests that it's a class with test cases, while it's not.

Reply:
Also do we define this, because LinearOperator directly doesn't work for older scipy versions? If so we should add a comment.